packages/graphql-testing/src/client.ts

```diff
@@ -1,67 +1,69 @@
-import {
-  ApolloClient,
-  SubscriptionOptions,
-  WatchQueryOptions,
-  ApolloClientOptions,
-  ObservableQuery,
-} from 'apollo-client';
+// import {
+//   ApolloClient,
+//   SubscriptionOptions,
+//   WatchQueryOptions,
+//   ApolloClientOptions,
+//   ObservableQuery,
+// } from '@apollo/client';
 
-type GetCurrentQueryResult = ApolloClient<
-  any
->['queryManager']['getCurrentQueryResult'];
+import {ApolloClient, ApolloClientOptions} from '@apollo/client';
 
-type AnyFetchPolicy =
-  | SubscriptionOptions['fetchPolicy']
-  | WatchQueryOptions['fetchPolicy'];
+// type GetCurrentQueryResult = ApolloClient<
+//   any
+// >['queryManager']['getCurrentQueryResult'];
+
+// type AnyFetchPolicy =
+//   | SubscriptionOptions['fetchPolicy']
+//   | WatchQueryOptions['fetchPolicy'];
 
 // no-cache and network-only fetch policies cause an initial async event to be
 // produced which is not properly caught inside of a react-testing act scope.
-const invalidFetchPolicies: AnyFetchPolicy[] = ['no-cache', 'network-only'];
```

Comment:
I temporarily switch the testing client for the real deal, I need to craft a test to figure out if this is still true.